projects/1.20/assets/vics-point-blank/pointblank/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+  "itemGroup.pointblank.items": "Point Blank",

```

```suggestion
  "itemGroup.pointblank.items": "搏枪斗械",
```
译名建议，因Point Blank在英文中指“(of a shot, bullet, or other [missile] fired from very close to its target.” 即（子弹，或其他弹射物在极近距离射向目标）。该译名之所以契合，是因为其中的“搏”“斗”部分暗示了近距离，而”枪“”械“部分扣及了本模组的主题，故而提出此建议。

---

我觉得这是个不错的译名